src/h5web/visualizations/line/LineToolbar.tsx

```diff
@@ -4,14 +4,38 @@ import Toggler from '../shared/Toggler';
 import { useLineConfig } from './config';
 
 function LineToolbar(): JSX.Element {
-  const [showGrid, toggleGrid] = useLineConfig(
-    state => [state.showGrid, state.toggleGrid],
+  const [
+    showGrid,
+    toggleGrid,
+    hasXLogScale,
+    toggleXLogScale,
+    hasYLogScale,
+    toggleYLogScale,
+  ] = useLineConfig(
+    state => [
+      state.showGrid,
+      state.toggleGrid,
+      state.hasXLogScale,
+      state.toggleXLogScale,
+      state.hasYLogScale,
+      state.toggleYLogScale,
+    ],
     shallow
   );
 
   return (
     <>
       <Toggler label="Show grid" value={showGrid} onChange={toggleGrid} />
+      <Toggler
+        label="X log scale"
+        value={hasXLogScale}
+        onChange={toggleXLogScale}
+      />
```

Comment:
A log scale on the x axis makes no sense at the moment, not until we implement axis-to-dimension mapping, so I would remove it from the toolbar. We'll have to show/hide the buttons in a smart way so users don't try to log an index-based axis.